Havit/Havit/Screens/Category/View/EditCategoryViewController.swift

```diff
@@ -15,6 +15,11 @@ class EditCategoryViewController: BaseViewController {
 
     // MARK: - property
 
+    var categoryId = 0
+    var titleText = ""
+    var iconImageId = 0
```

Comment:
p1;

init으로 사용 가능하기 때문에 
```suggestion
    var categoryId: Int
    var titleText: String
    var iconImageId: Int
```
이렇게 써줄 수 있을 거 같네요

Reply:
@YoonAh-dev  넵 반영하였습니다 ! fa6ea10

Reply:
@YoonAh-dev 혹싀..  오류가 나ㄴ느데... super.init() q부분에서 Property 'self.titleText' not initialized at super.init call 이렇게 뜹니댱

Reply:
반영꽈리  : c38a840